### PR TITLE
basic debian package building, may need revision

### PR DESCRIPTION
**Pull request by eslocombe**
feels kind of awkward using my name in the Maintainer for the control file. Maybe it should be something else.
This should probably be released under a new version/tag in which case, I will have to change a few of these files.

Tested only on a private pkg repo for trusty.

### CODE REVIEW

**Comment by lefred** [comment]
Hi Emily, 

Thank you for the patch, as you request it in your PR title, I will ask for a review before merging it.

thx already ;)

File: debian/files [hunk 1]
```diff
@@ -0,0 +1 @@
+innotop_1.11.4-1_amd64.deb database optional
```
Comment (bigon) [inline debian/files]: At first glance, this file shouldn't be committed in the repository

Comment (lefred) [inline debian/files]: Hi @bigon , what's the best practice then ?

Comment (bigon) [inline debian/files]: @lefred That file is supposed to be removed during the cleanup of the pkg, apparently there is a bug ATM not cleaning that file

So I would say that it should be removed and added to the .gitignore

**Comment by bigon** [comment]
A more global remark about this, it you want to have innotop to be included (again?) directly in debian, it's not a good idea to have that debian/ directory included upstream as the debian developers will need to remove it anyway.

Did you contact the "mysql" debian maintainer to see with them?

**Comment by eslocombe** [comment]
My intent was not to have it directly included in debian or ubuntu. I see innotop included in some mariadb client packages for ubuntu. My intent is to simply provide a .deb on a releases page. It is going to take me a bit more work to get to that point - see http://travis.debian.net (note, I am not currently using the debian travis integration, but that describes my intention)

**Comment by lefred** [comment]
Thank you for your contribution, let's make this available for the most and if distros want something following their standards even better, help is always appreciated ;)

**Pull request merged**